### PR TITLE
Create basic infra for maintainging server registry

### PR DESCRIPTION
**Pull request by pierewoj**
Testing:
* created a couple of containers, checked logs: `app_1 | [ScheduledExecutor RUNNING] INFO service.health.check.server.App$ScheduledExecutor - Currently there are 8 servers in the DB and 3 of them are active`

### CODE REVIEW

File: server/src/src/main/java/service/health/check/server/App.java [hunk 1]
```diff
@@ -1,67 +1,72 @@
...
-		for (service.health.check.models.Address address : addresses) {
-			AddressToCheck googleAddress = new AddressToCheck(address.getHost(),
-					address.getPort());
-			final ObjectMapper mapper = new ObjectMapper();
-			String messageJson = mapper.writeValueAsString(googleAddress);
-			channel.basicPublish("", Config.ADDRESSES_TO_CHECK_QUEUE,
-					MessageProperties.PERSISTENT_TEXT_PLAIN,
-					messageJson.getBytes());
-			logger.info("Server - Message sent: " + messageJson);
+			// servers that didn't perform a heatbreat for 10s are considered as inactive
```
Comment (js361014) [inline server/src/src/main/java/service/health/check/server/App.java]: heatbreat -> heartbeat

File: server/src/src/main/java/service/health/check/server/Database.java [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+    public List<Server> getAllServers() {
+        CriteriaBuilder criteriaBuilder = entityManager.getCriteriaBuilder();
+        CriteriaQuery<Server> criteriaQuery = criteriaBuilder.createQuery(
+                service.health.check.models.Server.class);
+        Root<Server> root = criteriaQuery.from(
+                service.health.check.models.Server.class);
+        criteriaQuery.select(root);
+        return entityManager.createQuery(criteriaQuery).getResultList();
+    }
```
Comment (js361014) [inline server/src/src/main/java/service/health/check/server/Database.java]: Możesz zrobić
```
public <T> List<T> getAll(Class<T> entityClass) {
        CriteriaBuilder criteriaBuilder = entityManager.getCriteriaBuilder();
        CriteriaQuery<T> criteriaQuery = criteriaBuilder.createQuery(entityClass);
        Root<T> root = criteriaQuery.from(entityClass);
        criteriaQuery.select(root);
        return entityManager.createQuery(criteriaQuery).getResultList();
    }
```

File: database-models/src/src/main/java/service/health/check/models/Server.java [hunk 3]
```diff
@@ -0,0 +1,58 @@
+package service.health.check.models;
+
+import org.hibernate.annotations.Type;
```
Comment (js361014) [inline database-models/src/src/main/java/service/health/check/models/Server.java:3]: Można usunąć.

File: server/src/src/main/java/service/health/check/server/App.java [hunk 4]
```diff
@@ -1,67 +1,72 @@
...
-import javax.persistence.EntityManager;
-import javax.persistence.criteria.CriteriaBuilder;
-import javax.persistence.criteria.CriteriaQuery;
-import javax.persistence.criteria.Root;
+import java.util.UUID;
+import java.util.concurrent.TimeUnit;
+import java.util.stream.Collectors;
 
 public class App {
+	private static final Server CURRENT_SERVER = new Server();
```
Comment (js361014) [inline server/src/src/main/java/service/health/check/server/App.java]: Jestem za currentServer https://stackoverflow.com/a/18640239

File: database-models/src/src/main/java/service/health/check/models/Server.java [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+
+	public void setId(String id) {
+		this.id = id;
+	}
+
+	public Instant getLastHeartbeat() {
+		return lastHeartbeat;
+	}
+
+	public void setLastHeartbeat(Instant host) {
```
Comment (js361014) [inline database-models/src/src/main/java/service/health/check/models/Server.java:55]: host -> lastHeartbeat

File: server/src/src/main/java/service/health/check/server/App.java (hunk 4 above)
Comment (pierewoj) [reply server/src/src/main/java/service/health/check/server/App.java]: zgoda https://google.github.io/styleguide/javaguide.html

**Pull request merged**